tests/wpt/web-platform-tests/dom/nodes/Node-cloneNode.html

```diff
@@ -27,25 +27,246 @@
+    create_element_and_check("rp",        HTMLElement);
+    create_element_and_check("rt",        HTMLElement);
+    create_element_and_check("ruby",      HTMLElement);
+    create_element_and_check("s",         HTMLElement);
+    create_element_and_check("samp",      HTMLElement);
+    create_element_and_check("script",    HTMLScriptElement);
+    create_element_and_check("section",   HTMLElement);
+    create_element_and_check("select",    HTMLSelectElement);
+    create_element_and_check("small",     HTMLElement);
+    create_element_and_check("source",    HTMLSourceElement);
+    create_element_and_check("spacer",    HTMLElement);
+    create_element_and_check("span",      HTMLSpanElement);
+    create_element_and_check("strike",    HTMLElement);
+    create_element_and_check("style",     HTMLStyleElement);
+    create_element_and_check("sub",       HTMLElement);
+    create_element_and_check("summary",   HTMLElement);
+    create_element_and_check("sup",       HTMLElement);
+    create_element_and_check("table",     HTMLTableElement);
+    create_element_and_check("tbody",     HTMLTableSectionElement);
+    create_element_and_check("td",        HTMLTableDataCellElement);
+    create_element_and_check("template",  HTMLTemplateElement);
+    create_element_and_check("textarea",  HTMLTextAreaElement);
+    create_element_and_check("th",        HTMLTableHeaderCellElement);
+    create_element_and_check("time",      HTMLTimeElement);
+    create_element_and_check("title",     HTMLTitleElement);
+    create_element_and_check("tr",        HTMLTableRowElement);
+    create_element_and_check("tt",        HTMLElement);
+    create_element_and_check("track",     HTMLTrackElement);
+    create_element_and_check("u",         HTMLElement);
+    create_element_and_check("ul",        HTMLUListElement);
+    create_element_and_check("var",       HTMLElement);
+    create_element_and_check("video",     HTMLVideoElement);
+    create_element_and_check("unknown",   HTMLUnknownElement);
+    create_element_and_check("wbr",       HTMLElement);
+}, "");
+
+test(function() {
+    var fragment = document.createDocumentFragment();
+    var copy = fragment.cloneNode();
+    check_copy(fragment, copy, DocumentFragment);
+}, "createDocumentFragment");
+
+test(function() {
+    var text = document.createTextNode("hello world");
+    var copy = text.cloneNode();
+    check_copy(text, copy, Text);
+    assert_equals(text.data, copy.data);
+    assert_equals(text.wholeText, copy.wholeText);
+}, "createTextNode");
+
+test(function() {
+    var comment = document.createComment("a comment");
+    var copy = comment.cloneNode();
+    check_copy(comment, copy, Comment);
+    assert_equals(comment.data, copy.data);
+}, "createComment");
+
 test(function() {
   var el = document.createElement("foo");
-  el.appendChild(document.createElement("bar"));
   el.setAttribute("a", "b");
   el.setAttribute("c", "d");
-  var c = el.cloneNode(false);
```

Comment:
I removed it because it looked redundant with [this test](https://github.com/servo/servo/pull/7070/files#diff-11d61e75a2bda690b78a49fd388779b6R266)